include/circt/Dialect/SV/SVStatements.td

```diff
@@ -879,3 +879,17 @@ def MacroDefOp : SVOp<"macro.def",
     MacroDeclOp getReferencedMacro(const hw::HWSymbolCache *cache);
   }];
 }
+
+
+//===----------------------------------------------------------------------===//
+// SV output control.
+//===----------------------------------------------------------------------===//
+
+def ReserveNamesOp : SVOp<"reserve_names", []> {
```

Comment:
I think the current implementation requires op to be in a top-level, correct? If so could you add `hasParent` here? I guess we could put the op in a HWModule op which affects only its local namespace but not sure it's worth allowing that.

Reply:
This is not something we need right now.